src/converter.js

```diff
@@ -168,7 +172,7 @@ converter._simple = (name, type_, node, mode) => {
 converter._visit = (name, node, mode='NULLABLE') => {
   let merged_node = node
   const ofs = ['allOf', 'anyOf', 'oneOf']
-  for(x=0 ; x<ofs.length; x++){
+  for(let x = 0; x < ofs.length; x++){
```

Comment:
A [`for...of`](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Statements/for...of) loop here would be more elegant.

Reply:
done